example/advection/advection_package.cpp

```diff
@@ -183,20 +184,15 @@ Real EstimateTimestep(Container<Real> &rc) {
   int ke = pmb->ke;
 
   Real min_dt = std::numeric_limits<Real>::max();
-  ParArrayND<Real> dx1("dx1", pmb->ncells1);
-  ParArrayND<Real> dx2("dx2", pmb->ncells1);
-  ParArrayND<Real> dx3("dx3", pmb->ncells1);
+  auto &coords = pmb->coords;
```

Comment:
For UniformCartesian, it's 3 ints + 3 std::array<Real, 3> + 1 Real.  Is there a problem with the reference?

Reply:
In general, I don't like reference variables because, like pointers, if the thing they're referencing goes away they can do weird things. In this case, it's certainly fine.

10 numbers is enough that it's probably worth using a reference, not doing a copy.